e2e-tsx/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "pfff",
```

Comment:
```suggestion
  "name": "foo-test",
```

Reply:
It was an onomatopoeia of breath because it was 11pm and I was losing my hair because I didn't know there was vrmt a way out with `findPackageJSON`.

Reply:
I know ;)